sql-cli/examples/_dags/simple.py

```diff
@@ -0,0 +1,82 @@
+# NOTE: This is an auto-generated file. Please do not edit this file manually.
+from airflow import DAG
+from airflow.utils import timezone
+from astro import sql as aql
+from astro.sql.table import Table
+
+with DAG(
+    dag_id="simple",
+    start_date=timezone.parse("2022-09-20"),
+) as dag:
+    a = aql.transform_file(
+        file_path="""
+            select 1
+        """,
+        parameters={
+        },
+        op_kwargs={
+            "output_table": Table(
+                name="a",
+            ),
+        },
+        task_id="a",
+    )
+    b = aql.transform_file(
+        file_path="""
+            select * from {{ a }}
```

Comment:
again, if we're using "transform_file", why are we passing in SQL statements instead of file paths?

Reply:
See https://github.com/astronomer/astro-sdk/pull/836#discussion_r978882933